### PR TITLE
Update values

### PR DESCRIPTION
**Pull request by hcarreras**
Even though we also live up to the previous values, I believe this way of looking at them is easier to understand and communicate.

### CODE REVIEW

File: behaviour_and_protocol/values.md [hunk 1]
```diff
@@ -1,23 +1,16 @@
...
-  - 4DWW
-  - Focus on carrer growth
-
-## Team over individual
-Practices:
-  - Collective product ownership
-  - Pair program
-  - Transparency
+## Collective contribution
+- Pairing. Teams create the
```
Comment (alexjeffburke) [inline behaviour_and_protocol/values.md:15]: @hcarreras truncated sentence here :)

Comment (hcarreras) [reply behaviour_and_protocol/values.md]: An issue when copying/pasting. Thanks for the heads up

File: behaviour_and_protocol/values.md [hunk 2]
```diff
@@ -1,23 +1,16 @@
-# Abtion values
+# Collective ownership
+Collective ownership gives control of the entire project to all team members. "Your part" is the product. Either it's a good product, or it is not. 
 
-## Solve real problems
-Practices:
-  - Focus on client value
-  - Simplicity
+## Collective vision
+- Open communication: No privileged information.
```
Comment (MarkusAbtion) [inline behaviour_and_protocol/values.md:5]: To follow the same structure through all of them:
```suggestion
- Open communication. No privileged information.
```

File: behaviour_and_protocol/values.md [hunk 3]
```diff
@@ -1,23 +1,16 @@
...
+- Break knowledge silos. No one is "the expert" in an area.
 
-## Long-term-value focus
-Practices:
-  - TDD
-  - Quality code base
-  - Knowledge sharing
+## Collective accountability
+- Work end-to-end. We need to be able to impact as much of the project as possible.
+- T-shape profiles. We still have roles, and they know about areas surrounding their expertise.
```
Comment (MarkusAbtion) [inline behaviour_and_protocol/values.md:11]: It seems like the adjective "T-shaped" is more common:
```suggestion
- T-shaped profiles. We still have roles, and they know about areas surrounding their expertise.
```

File: behaviour_and_protocol/values.md [hunk 4]
```diff
@@ -1,23 +1,16 @@
...
+- Integrated Q&A. The team takes direct responsibility for the work they've created.
```
Comment (MarkusAbtion) [inline behaviour_and_protocol/values.md:12]:
```suggestion
- Integrated QA. The team takes direct responsibility for the work they've created.
```

File: behaviour_and_protocol/values.md (hunk 3 above)
Comment (MarkusAbtion) [reply behaviour_and_protocol/values.md:11]: Could this be something like:
`T-shaped profiles. We each have strengths, but we understand areas outside of our expertise.`

**Review by MarkusAbtion — COMMENTED** [review]
This is a lot easier to read - and a lot easier to apply to daily work 💻

I only have a few wordings that I'd change.

File: behaviour_and_protocol/values.md (hunk 4 above)
Comment (jeppester) [reply behaviour_and_protocol/values.md:12]: In 5 years it will just be Q, and later it will probably be an emoji.

**Review by jeppester — APPROVED** [review]
I like this :+1: 

It allows a little more flexibility in how we work, and so better matches our day-to-day work were we iterate a lot on our work processes.

File: behaviour_and_protocol/values.md (hunk 4 above)
Comment (dj-abtion) [inline behaviour_and_protocol/values.md:12]:
```suggestion
- Integrated QA. The team takes direct responsibility for the work they've created.
```

I think Q&A means questions and answers, whereas QA means quality assurance.

File: behaviour_and_protocol/values.md [hunk 5]
```diff
@@ -1,23 +1,16 @@
...
-  - 4DWW
-  - Focus on carrer growth
-
-## Team over individual
-Practices:
-  - Collective product ownership
-  - Pair program
-  - Transparency
+## Collective contribution
+- Pairing. Team members work simultaneously on the same tasks.
```
Comment (dj-abtion) [inline behaviour_and_protocol/values.md:15]:
```suggestion
- Pair up! Team members work simultaneously on the same tasks.
```

File: behaviour_and_protocol/values.md (hunk 2 above)
Comment (dj-abtion) [inline behaviour_and_protocol/values.md:5]:
```suggestion
- Communicate openly. No privileged information.
```

Making this an imperative, more action-oriented.

File: behaviour_and_protocol/values.md [hunk 6]
```diff
@@ -1,23 +1,16 @@
-# Abtion values
+# Collective ownership
```
Comment (dj-abtion) [inline behaviour_and_protocol/values.md:1]:
```suggestion
Values are pointless if not put into practice. Our values are reflected in the following behaviors.

# Collective ownership
```

File: behaviour_and_protocol/values.md (hunk 3 above)
Comment (hcarreras) [reply behaviour_and_protocol/values.md:13]: I feel that this one needs another turn. I feel "strengths" is a too vague word. 
I'm going to keep it out of this PR, but feel free to open a new one or just go ahead, pair up with someone and improve the wording.

**Comment by hcarreras** [comment]
@jeppester I completely agree with the suggestion. Although, I find it hard to fit it into this current structure. 
Perhaps it could be another header like the one Daniel added.
We also have @KlausEnevoldsen-Abtion comment about focus on career development, and @bokh's philosophy on work-life balance is not reflected here either. 
There is definitely more work to do, but as you guys believe this is an improvement, I'll go ahead and merge it.

**Pull request merged**